infra-as-code/bicep/modules/managementGroups/managementGroups.bicep

```diff
@@ -1,5 +1,8 @@
 targetScope = 'tenant'
 
+@description('Optional Tenant Root Group ID. Used as top level management group parent when running WhatIf test.')
+param parTopLevelManagementGroupParentId string = ''
```

Comment:
@olljanat should we just set the default value to `tenant().tenantId`?

This way all customers by default will pass the what-if stage?

Reply:
Have provided suggestion that I have tested